react-components/src/actions/navigation/actions.js

```diff
@@ -90,3 +90,11 @@ export const showAccountCreated = ({ dispatch, t }) => {
     dispatchEvent(new CustomEvent('aem.accmg.step', { detail: { title: stepTitles[view](t) } }));
     dispatch({ type: 'changeView', view });
 };
+
+export const showView = ({ dispatch, t, view }) => {
+    const title = stepTitles[view](t);
+    if (title) {
```

Comment:
If `view` doesn't exist in the `stepTitles` object, there will already be an error thrown. So the `if (title)` is too late. Should be like:
```
let title = stepTitles[view];
if (!title) return;
title = title(t);
// ...
```

Reply:
Thanks. I've fixed it.